adafruit_si7021.py

```diff
@@ -177,6 +189,42 @@ def temperature(self):
         self._measurement = 0
         return value * 175.72 / 65536.0 - 46.85
 
+    @property
+    def heater_level(self):
+        """The heater level of the integrated resistive heating element.  Per
+        the data sheet, the levels correspond to the following current draws:
+
+        ============  =================
+        Heater Level  Current Draw (mA)
+        ============  =================
+        0             Off
+        1             3.09
+        2             9.18
+        3             15.24
+        .             .
+        5             27.39
+        .             .
+        9             51.69
+        .             .
+        17            94.20
+        ============  =================
+
+        """
+        if self._heater_enable_read:
+            return self._heater_level_read + 1
+        return 0
+
+    @heater_level.setter
+    def heater_level(self, level):
+        if not isinstance(level, int):
+            raise TypeError("Heater level must be int between 0 and 17, inclusive")
+        if not 0 <= level <= 17:
```

Comment:
According to the data sheet the heater setting has is only 4 bits so it must be 0-15
so the max in the setting should be 16 with the level-1